src/main/java/Task.java

```diff
@@ -0,0 +1,73 @@
+public class Task {
+    private String task;
+    private boolean isDone;
```

Comment:
variable is in camelCase and sound 'boolean'. Good!

Reply:
Thank you!